examples/ipython/printing-galgebra.ipynb

```diff
@@ -0,0 +1,396 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(cga3d.I())"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "xpdf(filename='test_latex_output.tex',paper=(9,10),pdfprog=None)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\r\n",
+      "\\documentclass[10pt,fleqn]{report}\r\n",
+      "\\usepackage[vcentering]{geometry}\r\n",
+      "\\geometry{papersize={9in,10in},total={8in,9in}}\r\n",
+      "\r\n",
+      "\\pagestyle{empty}\r\n",
+      "\\usepackage[latin1]{inputenc}\r\n",
+      "\\usepackage{amsmath}\r\n",
+      "\\usepackage{amsfonts}\r\n",
+      "\\usepackage{amssymb}\r\n",
+      "\\usepackage{amsbsy}\r\n",
+      "\\usepackage{tensor}\r\n",
+      "\\usepackage{listings}\r\n",
+      "\\usepackage{color}\r\n",
+      "\\usepackage{xcolor}\r\n",
+      "\\usepackage{bm}\r\n",
+      "\\usepackage{breqn}\r\n",
+      "\\definecolor{gray}{rgb}{0.95,0.95,0.95}\r\n",
+      "\\setlength{\\parindent}{0pt}\r\n",
+      "\\DeclareMathOperator{\\Tr}{Tr}\r\n",
+      "\\DeclareMathOperator{\\Adj}{Adj}\r\n",
+      "\\newcommand{\\bfrac}[2]{\\displaystyle\\frac{#1}{#2}}\r\n",
+      "\\newcommand{\\lp}{\\left (}\r\n",
+      "\\newcommand{\\rp}{\\right )}\r\n",
+      "\\newcommand{\\paren}[1]{\\lp {#1} \\rp}\r\n",
+      "\\newcommand{\\half}{\\frac{1}{2}}\r\n",
+      "\\newcommand{\\llt}{\\left <}\r\n",
+      "\\newcommand{\\rgt}{\\right >}\r\n",
+      "\\newcommand{\\abs}[1]{\\left |{#1}\\right | }\r\n",
+      "\\newcommand{\\pdiff}[2]{\\bfrac{\\partial {#1}}{\\partial {#2}}}\r\n",
+      "\\newcommand{\\lbrc}{\\left \\{}\r\n",
+      "\\newcommand{\\rbrc}{\\right \\}}\r\n",
+      "\\newcommand{\\W}{\\wedge}\r\n",
+      "\\newcommand{\\prm}[1]{{#1}'}\r\n",
+      "\\newcommand{\\ddt}[1]{\\bfrac{d{#1}}{dt}}\r\n",
+      "\\newcommand{\\R}{\\dagger}\r\n",
+      "\\newcommand{\\deriv}[3]{\\bfrac{d^{#3}#1}{d{#2}^{#3}}}\r\n",
+      "\\newcommand{\\grade}[1]{\\left < {#1} \\right >}\r\n",
+      "\\newcommand{\\f}[2]{{#1}\\lp{#2}\\rp}\r\n",
+      "\\newcommand{\\eval}[2]{\\left . {#1} \\right |_{#2}}\r\n",
+      "\\newcommand{\\Nabla}{\\boldsymbol{\\nabla}}\r\n",
+      "\\newcommand{\\eb}{\\boldsymbol{e}}\r\n",
+      "\\usepackage{float}\r\n",
+      "\\floatstyle{plain} % optionally change the style of the new float\r\n",
+      "\\newfloat{Code}{H}{myc}\r\n",
+      "\\lstloadlanguages{Python}\r\n",
+      "\r\n",
+      "\\begin{document}\r\n",
+      "\\begin{equation*} \\boldsymbol{e}_{1}\\wedge \\boldsymbol{e}_{2}\\wedge \\boldsymbol{e}_{3}\\wedge \\boldsymbol{e}\\wedge \\boldsymbol{e}_{{0}} \\end{equation*}\r\n",
+      "\\end{document}\r\n"
+     ]
+    }
+   ],
+   "source": [
+    "!cat test_latex_output.tex"
```

Comment:
This doesn't work on windows. Can we use something like:
```
with open("test_latex_output.tex") as f:
    print (f.read())
```

Reply:
OK. I forgot you had a similar modification earlier.

Reply:
Also we should somehow port our CI from Circle to Github Action to cover Windows.

Reply:
Perhaps, although it's barely worth it - the only parts that won't work on windows are tests like this, running the rest of the code on Windows is just unnecessary entropy being generated :)

Reply:
Yeah, but I believe there're other gains. Will experiment with it to see if it's worth it later.

Reply:
Let's leave that experimentation for a separate PR - I'd like to sort out my open printing PRs first, since I can't change some other stuff until the output is deterministic.